### ISSUE TITLE
Drawing is nice, can we also enter pre-defined geometry?

### ISSUE THREAD

**Issue by bcardarella** [PPLUSCHT/LBM#1]
For basic 2-d drawings it would be cool to have the option of entering in some pre-defined paths points that are drawn as a b-spline.

**Comment by PPLUSCHT** [comment]
Good idea! I thought about making the live draw cubic splines originally, but rushed the piecewise linear option through. I'll work on that when I get the chance.
